spec/bookingsync/api/client/bookings_spec.rb

```diff
@@ -117,4 +117,42 @@
         body: { bookings: [{ cancelation_reason: "payment_failed" }] }.to_json
     end
   end
+
+  describe ".add_bookings_fee_booking", :vcr do
+    let(:created_booking_id) {
+      find_resource("#{@casette_base_path}_create_booking/creates_a_booking.yml", "bookings")[:id]
+    }
+
+    it "adds bookings_fee" do
+      client.add_bookings_fee(created_booking_id, price: 100, times_booked: 2, name_en: "foobar")
```

Comment:
Not a fan of the `foobar` naming, way too abstract. A meaningful name would be best, which here could be `Cleaning Fee`